src/java_bytecode/jar_file.cpp

```diff
@@ -9,9 +9,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <cstring>
 #include <cassert>
 #include <unordered_set>
-
```

Comment:
I'd rather have more blank lines that fewer - is there a reason (such as IDEs doing automatic work) why this line is gone?